plugins/net_plugin/net_plugin.cpp

```diff
@@ -122,16 +123,16 @@ namespace eos {
         sync_received(),
         sync_requested(),
         socket( s ),
-        pending_message_size(),
-        pending_message_buffer(),
+        pending_message_size(0),
+        pending_message_buffer(1024*1024*4),
```

Comment:
how about we make a single constant, "ConnectionBufferSize" that is equal to 1024*1024*4 or whatever, rather than the magic numbers here and above.

Reply:
Will do.  Do we have a standard naming convention for constants?  I habitually name them all caps with an underscore between words, a carryover from C preprocessor days.  I can do initial caps camelcase if that's preferred.